docs/component-guides/pipelines.md

```diff
@@ -66,32 +67,15 @@ client = kfp.Client(host=f"https://{kubeflow_gateway_endpoint}/pipeline", cookie
 client.list_experiments(namespace=namespace)
 ```
 
-## S3 Access from Kubeflow Pipelines
-
-It is recommended to use AWS credentials to manage S3 access for Kubeflow Pipelines. [IAM Role for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) requires applications to use the latest AWS SDK to support the `assume-web-identity-role`. This requirement is in development, and progress can be tracked in the [open GitHub issue](https://github.com/kubeflow/pipelines/issues/3405).
-
-A Kubernetes Secret is required by Kubeflow Pipelines and applications to access S3. Be sure that the Kubernetes Secret has S3 read and write access.
+## AWS Access from Kubeflow Pipelines
 
-```
-apiVersion: v1
-kind: Secret
-metadata:
-  name: aws-secret
-  namespace: kubeflow
-type: Opaque
-data:
-  AWS_ACCESS_KEY_ID: <YOUR_BASE64_ACCESS_KEY>
-  AWS_SECRET_ACCESS_KEY: <YOUR_BASE64_SECRET_ACCESS>
-```
+User profiles can be granted permissions to access AWS resources. Pipelines under the profile namespace will have access to AWS resources as specified in the user profile's `awsIamRole`.
 
-- YOUR_BASE64_ACCESS_KEY: Base64 string of `AWS_ACCESS_KEY_ID`
-- YOUR_BASE64_SECRET_ACCESS: Base64 string of `AWS_SECRET_ACCESS_KEY`
+The instructions for managing AWS permissions for user profiles can be found [here](/kubeflow-manifests/docs/component-guides/profiles). The below [Example Pipeline](#example-pipeline) can be used to verify the configuration.
 
-> Note: To get a Base64 string, run `echo -n $AWS_ACCESS_KEY_ID | base64`
+### Example Pipeline
 
-### Example Pipeline 
-
-If you write any files to S3 in your application, use `use_aws_secret` to attach an AWS secret to access S3.
+In the below example, the `namespace` passed as a parameter to the `kfp_client` is the profile namespace. Change the `namespace` variable to match the profile namespace in which the pipeline will be deployed and ran in.
```

Comment:
you can extend that how role's permission will be inherited by kfp client.

Reply:
sure, assuming you mean boto3 client